roles/openshift_provisioning_aws_vpc/README.md

```diff
@@ -0,0 +1,65 @@
+OpenShift Provisioning AWS VPC
+=========
+
+This is a specialized role for provisioning AWS VPCs and is meant for advanced use only. The openshift_provisioning_aws role provides a simplified interface for basic and opinionated provisioning and should be preferred to this role.
+
+Requirements
+------------
+
+boto
+
+Role Variables
+--------------
+
+| Name                                   | Default value                                       |                     |
+|----------------------------------------|-----------------------------------------------------|---------------------|
+| openshift_provisioning_aws_vpc_id      | UNDEF                                               | Existing VPC ID to use |
```

Comment:
No, it is also used for short-circuiting the vpc creation to use an existing vpc. Idea being if the user specifies openshift_provisioning_aws_vpc_id, then we attempt to verify that it already exists and meets the config criteria that we have. If the user has permissions to query vpcs and the vpc config does not meet the criteria for OpenShift, then we will fail with an error message.